_posts/en/newsletters/2020-09-02-newsletter.md

```diff
@@ -0,0 +1,252 @@
+  potentially be used to swap out your Lightning channel counterparty or, in the
+  case of DLCs (see [Newsletter #81][news81 dlc]), swap out a portion of your
+  position without closing the channel. With various proposed Lightning channel
+  upgrades on the horizon, participants speculated about which upgrades could use dynamic commitments
+  (see [Newsletter #108][news108 dynamic commitments]) and which would require
+  [splicing][topic splicing]. Finally, Russell explained how the Lightning
+  protocol test suite lnprototest could be used to find bugs in existing
+  implementations and assist with ensuring interoperability across
+  implementations when building out new features.  ([transcript][sydney
+  transcript]).
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19797][] strays from Satoshi's original vision™ by
+  removing an obsolete validity check for IPv6 addresses constructed by
+  Bitcoin (Core) 0.2.8 and earlier.  Those early nodes had a bug that
+  would produce malformed addresses.
+  This change should have no impact on the current P2P network, as
+  affected versions are unable to communicate with current software due
+  to the introduction of checksums for P2P messages in Bitcoin
+  (Core) 0.2.9, which were later made mandatory.
+
+- [Bitcoin Core #19731][] extends the `getpeerinfo` RPC output with two new
+  fields: `last_block` and `last_transaction`. `last_block` is the time that
+  the peer sent the local node a block which the node had not seen
+  before and which passed preliminary validity checks. `last_transaction` is the
+  last time that the peer sent the local node a transaction that the node
+  accepted to its mempool. Both of these metrics are used when choosing peers
+  to disconnect, so it's useful for node operators to be able to access
+  the values.
+
+- [LND #4527][] adds a new `default-remote-max-htlcs` configuration
+  option that allows the user to specify the default maximum number of
+  unresolved HTLCs (payments) allowed in a channel.  This can minimize
+  the amount of onchain fees the user will have to pay if they need to
+  unilaterally close their channel, such as because of a fee ransom attack
+  (see [Newsletter #103][news103 fee ransom]).
+
+- [BIPs #982][] adds changes to [BIP340][] schnorr signatures (which
+  also affect [BIP341][] taproot).  BIP340 was previously
+  [changed][pk evenness update] to use two different tiebreakers for
+  conveying the Y coordinate of points: squaredness for the R point
+  inside signatures and evenness for the public keys (see [Newsletter #111][news111 proposed tiebreaker]).  As
```

Comment:
This link to Newsletter 111 doesn't cover the change to use two different tiebreakers. Did you mean to link to https://bitcoinops.org/en/newsletters/2020/02/05/#alternative-x-only-pubkey-tiebreaker ?

Reply:
Link was correct but its position in the paragraph was wrong.  Moved.